src/ronny.js

```diff
@@ -38,7 +38,7 @@ bot.on('message', function(data) {
               if (typeof message === 'function')
                 message = message();
               if (message.params) {
-                params = message.params;
+                for (var attrname in message.params) { params[attrname] = message.params[attrname];}
```

Comment:
I think that should work for most cases. 
Attachments are the only nested values for the slack api parameters.